sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -439,7 +442,23 @@ private void appendListOfFields(List<ImportColumn> importColumns, Class<?> clazz
 				continue;
 			}
 
-			Method readMethod = null;
```

Comment:
Instead of writing a new implementation for this, maybe you could use the `CountryFieldVisibilityChecker` which is already available in the `sormas-api` project

Reply:
nice